### ISSUE TITLE
Only the @latest should be pushed to brew

### ISSUE THREAD

**Issue by tamalsaha** [cue-lang/cue#1588]
Hello, 
I have cue cli installed via brew. Yesterday when I ran `brew update; brew upgrade` , I got the new cue v0.4.3-beta.1 . Yay, for the new release.

But I think non GA releases should not be pushed to the brew, as there may be bugs in beta releases etc.

Thanks!

**Comment by verdverm** [comment]
https://github.com/Homebrew/brew/issues/3808

This seems like something brew should handle via semver comparison. Not sure if they ever fixed the above issue or if CUE needs to adjust the formula so homebrew behaves as expected.

fwiw, go install ...@latest handles these situations. It would seem onerous to put this on every package maintainer who uses semver correctly.

**Comment by myitcv** [comment]
I've just updated the title to reflect the more general case of problems associated with new releases.

I've also opened https://github.com/goreleaser/goreleaser/discussions/3114 to ask for guidance on how best to solve this issue within GoReleaser.

Note the problem also applies to Docker image builds, specifically the updating of the `:latest` tag.

Depending on the answer to that question we can either:

* fix our GoReleaser configuration or
* maintain our GoReleaser configuration in CUE, and make the Homebrew Tap and Docker config conditional on some injected value that determines whether we are releasing an `@latest` version or not.
